views/search.hbs

```diff
@@ -0,0 +1,36 @@
+<link rel="stylesheet" href="/css/search.css">
+{{> nav}}
+
+{{#if users}}
+    <div class="container mt-5">
+        <div class="row">
+            <div class="col-lg-12 col-sm-12 col-12 main-section">
+                <div class="row text-center mb-3">
+                {{#each users as |user|}}
+                    <a href = '/profile/others/{{user._id}}' class="col-lg-4 col-sm-4 col-12">
+                        <div class="row-fluid section">
+                            <div class="col-lg-12 col-sm-12 col-12">
+                                <img src='{{user.profilePic}}'>
+                                <h1>{{user.firstName}} {{user.lastName}}</h1>
+                                <p>@{{user.username}}</p>
+                            </div>
+                            <div class="col-lg-12 col-sm-12 col-12">
+                                <div class="row user-detail">
+                                    <div class="col-lg-6 col-sm-12 col-6 text-left">
+                                        <h2><i class="fa fa-envelope" aria-hidden="true"></i>{{user.email}}</h2>
+                                    </div>
+                                    <div class="col-lg-6 col-sm-12 col-6 text-right">
+                                        <h2><i class="fa fa-phone" aria-hidden="true"></i>{{user.mobile}}</h2>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </a>
+                {{/each}} 
+                </div>
+            </div>
+        </div>
+    </div>
+    {{else}}
+        <h1 style = "color:white;" class = 'display-4'>No user with that username has been found.</h1>
+{{/if}}
```

Comment:
Ang lalim. That's the reason why we're enforcing One Tab == 2 space setting.

Reply:
Noted. I converted all the one tab 4 space files into one tab 2 space files.